packages/driver/src/dom/animation.ts

```diff
@@ -0,0 +1,21 @@
+import $ from 'jquery'
```

Comment:
Could we do this without jquery?

Reply:
I didn't investigate changing the implementation since I was just moving the code around. Possibly something for the future though.